source/guide_roundcube.rst

```diff
@@ -0,0 +1,222 @@
+  * Database server (use ``localhost``)
+  * Database name (e.g. ``isabell_roundcube``)
+  * MySQL username (equals your Uberspace username, e.g. ``isabell``)
+  * MySQL password - you should know your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>` by now. If you don't, start reading again at the top.
+  * db_prefix (e.g. ``rc_``)
+
+
+
+IMAP & SMTP Settings
+--------------------
+
+  * default_host (use *ssl://„server“.uberspace.de*, e.g. ``ssl://stardust.uberspace.de``)
+  * default_port (use ``993``)
+  * smtp_server (use *tls://„server“.uberspace.de*, e.g. ``tls://stardust.uberspace.de``)
+  * smtp_port (use ``587``)
+  * junk_mbox (use ``Spam``)
+
+.. warning:: Make sure to add the prefixes (``ssl://`` / ``tls://``) as mentioned above, otherwise you won't be able to receive or sent mails. Uberspace does not support :manual_anchor:`insecure access <mail-access.html#client-settings>`.
+  
+.. note:: We'll adjust the default setting ``Junk`` to ``Spam`` to refer to the default :manual_anchor:`default Spamfolder <mail-spam.html#configure-spam-folder>`.
+ 
+
+``Activate`` the option *Use the current IMAP username and password for SMTP authentication*.
+
+Display settings & user prefs
+-----------------------------
+
+Enter your desired language for RoundCube under **language**.
+The code for German is ``de_DE``, English is ``en_US``, the code for French is ``fr_FR`` etc
+
+.. note:: Users can set their own language.
+
+Plugins
+-------
+
+If you want to add Plugins to your configuration tick the corresponding checkboxes.
+  
+When you're done click *CONTINUE*. The configuration file will be created and stored in ``/var/www/virtual/$USER/html/config``.
+
+Finishing installation
+======================
+
+On the last tab *Test config* click on **initialize database** to initialize the database.
+
+.. note:: Optionally you can check your SMTP & IMAP config by entering valid credentials down here, but it's not necessary.
+
+Your done. Point your Browser to your installation URL ``https://isabell.uber.space`` and
+admire your shiny new webmail client!
+
+
+Best practices
+==============
+
+Security
+--------
+
+.. warning:: For security reasons, you should remove the installer-directory from the source files.
+
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html
+ [isabell@stardust html]$ rm -rv installer/
+ [isabell@stardust html]$
+ 
+Login Pattern
+-------------
+
+Use your full email address and the corresponding password to login. This works with your ``@uber.space`` address as well as with any addresses using your :manual_anchor:`own domains <mail-domains.html#mail-domains>`.
+
+
+
+Updates
+=======
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+If there is a new version available, **always make a backup first**.
+Refer to the Upgrade_ Manual of the official Roundcube Wiki for further instructions.
```

Comment:
Please outline the easy `./bin/installto.sh` updating method here. It's fine to link to the more complicated manual way for more information.

Reply:
i added the update-section to the guide.